tests/docker/main.yml

```diff
@@ -114,6 +114,7 @@
 
   pre_tasks:
     - block:
+      when: g_docker_latest
```

Comment:
This was a failure in the log.  I'm not sure how this syntax will work here.  I would try
```
- when: g_docker_latest
  block:
    - name: ...
```